src/System.Management.Automation/engine/Types_Ps1Xml.cs

```diff
@@ -1380,6 +1382,8 @@ public static IEnumerable<TypeData> Get()
             td167.Members.Add("IPV6Address",
                 new ScriptPropertyData(@"IPV6Address", GetScriptBlock(@"$iphost = [System.Net.Dns]::GetHostEntry($this.address)
           $iphost.AddressList | Where-Object { $_.AddressFamily -eq [System.Net.Sockets.AddressFamily]::InterNetworkV6 } | Select-Object -first 1"), null));
+            td167.Members.Add("Destination",
+                new AliasPropertyData("Destination","Address"));
```

Comment:
Same here. Do we still need this change?

Reply:
I don't use Win32_PingStatus - we can remove this here.

Reply:
removed